addon/styles/addon.scss

```diff
@@ -74,7 +74,7 @@ $frost-modal-overflow-shadow-color: rgba(0, 0, 0, .13);
       margin-left: 10px;
     }
 
-    &-text {
```

Comment:
@sandersky I can add the -text back and put the TODO to remove.  I mean I'm already in it...